InquirerPy/base.py

```diff
@@ -186,12 +186,14 @@ def _get_prompt_message(
         """
         display_message = []
         if self.status["result"] == INQUIRERPY_KEYBOARD_INTERRUPT:
-            display_message.append(
-                ("class:skipped", "%s %s " % (self._qmark, self._message))
-            )
+            question = "%s " % self._qmark if self._qmark != "" else ""
+            question += "%s " % self._message
```

Comment:
These 2 lines don't really change any behaviour if `self._qmark = ""`. It still results in `%s %s`. You might wanna do the same logic as the bottom change `if self._qmark != "":`.

Thanks